### DISCUSSION TITLE
Avoid excessive Dom Size

### DISCUSSION THREAD

**Discussion by MisherLiu** [UI5/webcomponents#3735] [Q&A]
Hi Dev Team,
I use ui5-table in a Vue demo, however after run the Lighthouse report in Chrom devtool, there is an issue which is "avoid excessive dom size", this impact on the performance, could you look into this? Is there anything I use wrong way?

![image](https://user-images.githubusercontent.com/64764287/130931195-c3083889-d36e-4be1-82bc-728522f35444.png)

Sample code snippet:
![image](https://user-images.githubusercontent.com/64764287/130931588-2dc1468c-1261-4ccb-a6e2-5c03a342353e.png)

**Comment by Roba1993** [comment]
Yeah same here, 30k dom nodes for the ui5-datetime-picker component makes this really hard to use. I will create an issue for this and hope it will be fixed.
